.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+hint-report/
+dist/
```

Comment:
- [ ] It would be better if remove `dist` from the node modules.

> **Example of dist folder structure:**

![r1](https://user-images.githubusercontent.com/70483835/167737673-d64f3f27-4681-415a-ac8a-36527df5c5a4.png)

Reply:
Hi, thank you for your feedback!!
I made this change but now I'm facing lighthouse errors related to the files in the dist folder.

Reply:
Hi @Trictonicmp, kindly include the dist folder. It is essential to have the `dist` folder in the root folder because it is your project's minified and concatenated version. When you run `npm run build`, all the codes in the `src` folder get concatenated to the dist folder which is used for production. You can share the lighthouse issue or Google the cause of the lighthouse linter check failing to pass. 👍